services/rclone/queue.go

```diff
@@ -0,0 +1,56 @@
+package rclone
+
+import (
+	"github.com/ansel1/merry/v2"
+	"sync"
+	"time"
+)
+
+const maxConcurrentTransfers = 5
+
+var queueLock = sync.Mutex{}
+var transferQueue = map[Priority][]chan bool{}
+
+func init() {
+	for _, priority := range Priorities.Members() {
```

Comment:
this `init` seems to never be called. But maybe it doesnt need to?

Reply:
Init is called when package is loaded.